libraries/AP_HAL_Linux/Util.cpp

```diff
@@ -28,6 +28,8 @@ void Util::init(int argc, char * const *argv) {
     saved_argc = argc;
     saved_argv = argv;
 
+    _system_time_was_set = CONFIG_HAL_BOARD == HAL_BOARD_LINUX;
```

Comment:
Is this redundant? We are in Linux namespace.

Reply:
Check my last comment.
https://github.com/ArduPilot/ardupilot/pull/7716#discussion_r169719140